### ISSUE TITLE
[MU4 Issue] Regression: master tuning setting not implemented

### ISSUE THREAD

**Issue by fredg999** [musescore/MuseScore#15566]
**Describe the bug**
The "master tuning" setting, which was located in the Synthesizer window in MuseScore 3, is not yet implemented in MuseScore 4.

**To Reproduce**
1. Open MuseScore 4.
2. Search everywhere for a master tuning setting.
3. Notice that you cannot find the setting.

**Expected behavior**
It should be possible to change the master tuning in MuseScore 4.

**Screenshots**
It's hard to show what's not there! ;)
See MuseScore 3's handbook to see how the setting was previously available: https://musescore.org/en/handbook/3/synthesizer#tuning

**Platform information**
- OS: Ubuntu 22.04.1 LTS, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-223580304, revision: github-musescore-musescore-d759f29

**Additional context**
Since this issue probably needs design, I'd like to suggest that master tuning should be a text element to be found in the Pitch palette, and this for several reasons.

First, it would allow people reading the score to know the master tuning intended by the composer, in a similar way that tempo texts work. The text element could always be made invisible in case a user only wants the playback effect, but no text on the score.

Second, it would make it easier to allow multiple master tunings in a single score, both horizontally (change over the course of a piece; e.g. Jacob Collier's cover of "In the Bleak Mid-Winter") and vertically (instruments with different master tunings; e.g. scores for two pianos tuned a quarter-tone apart). By the way, a master tuning line akin to the accel. and rit. lines would be awesome (and I've seen some people request them on the forums), but I understand that's a relatively low-priority concern.

Third, while most people define concert pitch relative to A4, there are a few occasions where choosing another reference note could be useful. For example, scientific pitch is defined as C4 = 256 Hz. Using a text element for master tuning means there's room to make MuseScore recognized the pitch name given in the text and calculate the master tuning from that note. Of course, A4 would be picked by default if MuseScore cannot parse successfully the pitch name in the text.

To sum up, I think text elements for master tuning would be more intuitive and flexible for both beginners and advanced users, compared to setting it in the mixer or any other equivalent to the old Synthesizer window.

**Comment by DaffodilAura** [comment]
This indeed could be part of how MuseScore natively supports microtonality in the future.  I'll have to add this information to my document...

**Comment by 4ldo** [comment]
... probably with the exception of grand pianos and church organs, the first thing a musician does with an instrument is to tune it.
This is a basic functionality

**Comment by zorba77** [comment]
A lot of basic functions are not ported to Musescore 4. The lack of master tuning setting make MU4 unuseable for ancient music, for example.

**Comment by MarcSabatella** [comment]
That's not true - if you want everything tuned lower (or higher), simply select all (Ctrl+A) then use the Playback section of the Properties panel to set whatever tuning adjustment you like in cents.  In many ways this is better than the old system-specific master tuning setting, since that wouldn't allow others to hear the score as you are.  But indeed, someday it might be nice to have both options.

**Comment by zorba77** [comment]
Example: 
- I wrote a piece
- I select all notes and change the pitch (I halve also to calculate the cents of difference instead of setting simply 415hz)
- I made some change to the score... I have to change the pitch for every notes I edited, or select every note. 
- In the meanwile I have some note at 440hz and others at 415hz
This is very confusing
In Europe orchestra are tuned at 442hz, why I cannot have MS set directly to 442hz?
Also for the lack of this function I need to use MS 3.7

**Comment by MarcSabatella** [comment]
Indeed, you are welcome to *choose* to use an older version if you prefer to keep your tuning adjustments secret and prevent others from hearing them when you share the score.  But  you do still have the option of using the mu4 tuning adjustments that works more permanently.  Yes, you need to calculate cents (easy to look up online), and yes you need to update it after editing (takes but seconds to update the whole score).

And yes, again, it will be nice to someday have the option of having the secret private unshareable master tuning adjustment in MU4 too.  I’m just trying to help you *now*.

**Comment by zorba77** [comment]
I do not want to keep secret my tuning (ancient tuning is a secret?) I want the possibility to work with a different pitch on my computer.
And I need this possibility now, so I have to use MS 3.7 (or another software)

**Comment by MarcSabatella** [comment]
I mean, by using this option, you are keeping your tuning a secret from those you share the score with, since that setting won't be honored when others access your score.  It's private only to you when playing the score on your own system.  If that's OK, and then by all means, you are free to choose to use an older version of MuseScore rather than choosing to use the per-note tuning feature in MU4 which is more pervasive.

**Comment by 4ldo** [comment]
I agree with zorba77 in addition to the ancient music use case I would argue that, for example in Europe, everybody tunes A4=442 Hz.
The method described by Marc may work, but is cumbersome and requires additional steps and calculations. 
I you want to use Musecsore as an instrument an easy, standard (using Hz instead of cents) is needed
The idea is to give every musician the possibility to generate a pitch that matches the pitch of other instruments she/he is using. 
Yes, the master tuning is, not only, individual, but may change, for the same person, depending on the group of instruments I'm working with

**Comment by cbjeukendrup** [comment]
Consolidating this as a duplicate into the more concise https://github.com/musescore/MuseScore/issues/18257. Feel free to react to that issue with a 'thumbs up' emoji (_not_ a comment), by way of 'upvoting' it.
